### ISSUE TITLE
Change hard-coded cache fetch in front-end image to dynamic for "most recent"

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#56146]
## Issue Description
While figuring out the front end image for PEs, we hard-coded telling the container to pull a specific cached content-build file. We'll need to change this to be dynamic, fetching the most recent cached build each time.

---
## Tasks
- [ ] Update front end container launch script in vets-website to pull the most recent content-build version instead of the hard-coded one being used prior.

## Acceptance Criteria
- [ ] The newest version of content build is always included with each PE.
---

**Comment by CBonade** [comment]
I have a solution ready to go for this, however, I lack the permissions to view this S3 bucket. There have been multiple attempts prior to get my permissions adjusted for this but so far, none have succeeded. 

Have a support thread open currently waiting to be unblocked so I can list the files.

**Comment by CBonade** [comment]
![image.png](https://images.zenhubusercontent.com/60aa993146d844d86f494162/a965ea7c-be76-44b9-a1cb-59883d27f778)

Since my personal access got sorted out I was able to verify that the solution I had, works. However, the GHA bot still does not have permissions to do this so, unfortunately, this is blocked until it does.

**Comment by CBonade** [comment]
Got this done at the nth hour of the sprint. Ran into a number of complications with where this environment variable needed to be defined and how to get this to persist when it ships off to ECR, but still be dynamic. We are now fetching the most recent cache for content-build and packaging in the production version of content, with each image build in vets-website. Workflow still commented out until we are ready to start spinning up preview environments, but I did test to make sure the workflow ran properly and it had no issues. Also ran the docker container locally to make sure it worked with the variable being passed.
